jujugui/static/gui/src/app/components/scale-service/scale-service.js

```diff
@@ -130,10 +130,16 @@ YUI.add('scale-service', function() {
           <div className="scale-service--units">
             <input
               className="scale-service--units__input"
-              type="text"
+              type="number"
+              autoComplete="off"
               name="num-units"
               onChange={this._updateState}
-              ref="numUnitsInput"/>
+              ref={function(input) {
+                if (input != null) {
+                  input.focus();
+                  return "numUnitsInput";
```

Comment:
This is quite interesting, for this to work that means that the ref is calculated after the element is in the DOM. Good to know.